.github/workflows/update-openapi.yml

```
@@ -0,0 +1,168 @@
+name: Update OpenAPI Specification
+
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)